SpherePacking/ModularForms/iteratedderivs.lean

```diff
@@ -17,7 +17,7 @@ theorem upper_ne_int (x : ℍ) (d : ℤ) : (x : ℂ) + d ≠ 0 :=
   by
   by_contra h
   rw [add_eq_zero_iff_eq_neg] at h
-  have h1 : 0 < (x : ℂ).im := by simp only [coe_im]; exact im_pos x
+  have h1 : 0 < (x : ℂ).im := by simp; exact im_pos x
```

Comment:
I think we should keep the `simp only` version to avoid having a non-terminal `simp`. (On a side note, we have a few non-terminal `simp`s scattered around the repo that we need to get rid of at some point.)